mantidproject/mantid
# Remove `CONDA_ENV` variable from CMake

**Description of work**
This PR removes the `CONDA_ENV` variable from the Mantid CMakeLists. This variable is now superfluous as the only supported way to build Mantid now is through the use of Conda.

The `CONDA_BUILD` variable is still needed because this is used to specify whether to build a package using the conda build tool. This is usually false when developing locally, and true on the various mantid pipelines.

**To test:**
Make sure the tests pass and do a code review
Make sure this pipeline job is successful
https://builds.mantidproject.org/job/build_packages_from_branch/387/

Fixes #35304 

#### Reviewer ####

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

#### Gatekeeper ####

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
